packages/adblocker-extended-selectors/src/index.ts

```diff
@@ -6,7 +6,7 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-export { parse, tokenize } from './parse.js';
+export { parse, tokenize, walk } from './parse.js';
```

Comment:
"walk" does not appear to be used in this file.

Reply:
it is exported for external use